<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormUtilities</class>
 <widget class="QWidget" name="FormUtilities">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1206</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Utilities</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qBorgPilot.qrc">
    <normaloff>:/images/borgpilot_16x16.png</normaloff>:/images/borgpilot_16x16.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" rowspan="2">
    <widget class="QFrame" name="frame_Search">
     <property name="maximumSize">
      <size>
       <width>500</width>
       <height>270</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_Name">
          <property name="maximumSize">
           <size>
            <width>87</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Script Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ComboBoxScriptName_P" name="comboBox_ScriptName"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item alignment="Qt::AlignmentFlag::AlignTop">
         <widget class="QLabel" name="label_Repos">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Repositorioes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ListViewRepos_P" name="listView_Repos">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::EditTrigger::CurrentChanged|QAbstractItemView::EditTrigger::SelectedClicked</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
          </property>
          <property name="resizeMode">
           <enum>QListView::ResizeMode::Adjust</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="groupBox_Commands">
     <property name="maximumSize">
      <size>
       <width>700</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Commands</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_Info">
        <property name="text">
         <string>Info</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="5">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="radioButton_Cmd_Delete">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="radioButton_Cmd_Prune">
        <property name="text">
         <string>Prune</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_List">
        <property name="text">
         <string>List</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="4">
       <widget class="ListWidget_Delete_P" name="listWidget_Delete">
        <property name="modelColumn">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3" rowspan="4">
       <widget class="ListWidget_Prune_P" name="listWidget_Prune">
        <property name="modelColumn">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radioButton_Check">
        <property name="text">
         <string>Check</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="radioButton_Diff">
          <property name="text">
           <string>Diff</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="radioButton_Rename">
          <property name="text">
           <string>Rename</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Results</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="toolButton_ClearResults">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::EditClear"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_SaveAsResults">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::DocumentSaveAs"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1099</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit_Command">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If there are placeholders, replace them with their corresponding ones: e.g.: REPO::{hostname}-{user} with REPO::backup-user01&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="maxLength">
           <number>500</number>
          </property>
          <property name="placeholderText">
           <string>placeholder</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_Parameters">
          <property name="maxLength">
           <number>1024</number>
          </property>
          <property name="placeholderText">
           <string>Argument list</string>
          </property>
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_Exec">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Execute the command&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QTextEdit" name="textEdit_Results">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 85, 0);</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextInteractionFlag::TextSelectableByKeyboard|Qt::TextInteractionFlag::TextSelectableByMouse</set>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBoxScriptName_P</class>
   <extends>QComboBox</extends>
   <header location="global">comboboxscriptname_p.h</header>
  </customwidget>
  <customwidget>
   <class>ListViewRepos_P</class>
   <extends>QListView</extends>
   <header location="global">listviewrepos_p.h</header>
  </customwidget>
  <customwidget>
   <class>ListWidget_Delete_P</class>
   <extends>QListWidget</extends>
   <header location="global">listwidget_delete_p.h</header>
  </customwidget>
  <customwidget>
   <class>ListWidget_Prune_P</class>
   <extends>QListWidget</extends>
   <header location="global">listwidget_prune_p.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBox_ScriptName</tabstop>
  <tabstop>listView_Repos</tabstop>
  <tabstop>radioButton_Info</tabstop>
  <tabstop>radioButton_List</tabstop>
  <tabstop>radioButton_Check</tabstop>
  <tabstop>radioButton_Diff</tabstop>
  <tabstop>radioButton_Rename</tabstop>
  <tabstop>radioButton_Cmd_Delete</tabstop>
  <tabstop>listWidget_Delete</tabstop>
  <tabstop>lineEdit_Command</tabstop>
  <tabstop>toolButton_Exec</tabstop>
  <tabstop>toolButton_ClearResults</tabstop>
  <tabstop>toolButton_SaveAsResults</tabstop>
  <tabstop>textEdit_Results</tabstop>
 </tabstops>
 <resources>
  <include location="qBorgPilot.qrc"/>
 </resources>
 <connections/>
</ui>
